go/apps/api/run.go

```diff
@@ -234,6 +214,54 @@ func Run(ctx context.Context, cfg Config) error {
 		DB:     db,
 	})
 
+	// Initialize cache invalidation topic
+	var cacheInvalidationTopic *eventstream.Topic[*cachev1.CacheInvalidationEvent]
```

Comment:
can we initialize this with a NOOP instead of nil?

Reply:
hmm, I see we do a nil check in the cache

I just ran into so many issues with stuff like this where it might or might not be nil.
It's convenient to know that it's always available, even if just noop